### PR TITLE
Sora Card tests refactoring

### PR DESCRIPTION
**Pull request by pg-vlad**

### CODE REVIEW

File: src/main/java/pages/account/VerifyPhoneNumberOtpPage.java [hunk 1]
```diff
@@ -2,22 +2,40 @@
...
+    private SelenideElement selectCountryField;
+    @AndroidFindBy(xpath = "/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.FrameLayout/androidx.compose.ui.platform.ComposeView/android.view.View/android.view.View/android.widget.ScrollView/android.widget.EditText[2]")
+    @iOSXCUITFindBy(iOSNsPredicate = "value == \"Phone Number\"")
+    private SelenideElement phoneNumberField;
 
     @AndroidFindBy(xpath = "/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.FrameLayout/androidx.compose.ui.platform.ComposeView/android.view.View/android.view.View/android.widget.ScrollView/android.widget.EditText")
     @iOSXCUITFindBy(iOSNsPredicate = "value == \"Phone Number\"")
```
Comment (Aelikka) [inline src/main/java/pages/account/VerifyPhoneNumberOtpPage.java:27]: this locator looks incorrect

File: src/main/java/pages/account/VerifyPhoneNumberOtpPage.java [hunk 2]
```diff
@@ -2,22 +2,40 @@
...
-    private SelenideElement phoneNumberOTPField;
+    private SelenideElement otpField;
+
+    @AndroidFindBy(xpath = "/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.FrameLayout/androidx.compose.ui.platform.ComposeView/android.view.View/android.view.View/android.view.View[2]/android.widget.Button[2]")
+    private SelenideElement searchCountryField;
```
Comment (Aelikka) [inline src/main/java/pages/account/VerifyPhoneNumberOtpPage.java:31]: it's not a field, it's a button. rename, pls

File: src/main/java/pages/account/VerifyPhoneNumberOtpPage.java [hunk 3]
```diff
@@ -2,22 +2,40 @@
...
+
+    @AndroidFindBy(className = "android.widget.EditText")
+    private SelenideElement androidWidgetEditText;
```
Comment (Aelikka) [inline src/main/java/pages/account/VerifyPhoneNumberOtpPage.java:34]: this is not a good name for element, rename, pls

File: src/test/resources/test.properties [hunk 4]
```diff
@@ -2,11 +2,12 @@ mnemonic1=humble poem better wheat work unhappy penalty company steak material c
...
+phonePending =23456782273
+phoneFailed =23456782290
+phoneRejected =23456782268
 testOtp=123456
 referrerLink1 =polkaswap.io/#/referral/cnRY1Xy3Mtbw3xS6XYcm7T4UFGC6k4wzfHCxEydrGZ2ssQS8S
 referrerLink2=polkaswap.io/#/referral/cnUXQmtmidknFFyZFvMdFJuzUY7eB91y9ygEVTzibWZZCmZb9
-assetId=cnUXQmtmidknFFyZFvMdFJuzUY7eB91y9ygEVTzibWZZCmZb9
\ No newline at end of file
+assetId=cnUXQmtmidknFFyZFvMdFJuzUY7eB91y9ygEVTzibWZZCmZb9382
+countryCode=Montenegro
```
Comment (Aelikka) [inline src/test/resources/test.properties:13]: not sure we should put countries in config. it doesn't depend on environment or a user. maybe just a variable in pageobject will do for now.

**Review by Aelikka — CHANGES_REQUESTED** [review]

File: src/test/resources/test.properties (hunk 4 above)
Comment (pg-vlad) [reply src/test/resources/test.properties:13]: We have phone numbers of different countries in properties

**Review by Aelikka — APPROVED** [review]

**Pull request merged**